src/components/Modals/ModalChangePost.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <ModalWrapper
+        :is-open="isOpen"
+        @close="onClose"
+    >
+        <div
+            class="modal-inner"
+            @click="$event.stopPropagation()"
+        >
+            <input
+                v-model="titleInputValue"
+                type="text"
+            />
+            <textarea
+                v-model="bodyInputValue"
+                type="text"
+            />
+            <button @click="onChangePostData">Change data</button>
+        </div>
+    </ModalWrapper>
+</template>
+
+<script setup lang="ts">
+import { usePostsStore } from '@/stores';
+import { ModalWrapper } from '@/components';
+
+const { isOpen, postId, title, body } = defineProps<{
+    isOpen: boolean;
+    postId: number | string;
```

Comment:
only number

Reply:
not only, because i use function `uid`, it returns string, but also i get ids with type number from server. is it reasonable to change the type?